### PR TITLE
small stuff, sync with qs

### PR DESCRIPTION
**Pull request by sezero**
- remove rest of the SDL1 refs from source
- remove unnecessary _WIN32 checks from windows source
- warning fixes for bintoc.c
- sync with quakespasm-0.92.0: (missed parts in SCR_ModalMessage() and R_BuildLightMap()
- sync with quakespasm tree: rest of the small stuff, for sake of symmetry

### CODE REVIEW

File: Quake/Makefile [hunk 1]
```diff
@@ -70,7 +70,6 @@ define do_strip
 	$(call cmd_strip,$(1));
 endef
 endif
-CFLAGS += -std=gnu99
```
Comment (ProfessorKaos64) [inline Quake/Makefile]: Seems to compile fine on Arch Linux. What distro did you test this change on?

Comment (sezero) [inline Quake/Makefile]: On 9/13/16, ProfessorKaos64 <notifications@github.com> wrote:
>> @@ -70,7 +70,6 @@ define do_strip
>>  	$(call cmd_strip,$(1));
>>  endef
>>  endif
>> -CFLAGS += -std=gnu99
>
> Seems to compile fine on Arch Linux. What distro did you test this change
> on?

Fedora 20.

Comment (Novum) [inline Quake/Makefile]: Are you sure this is fine? There are still plenty of functions where I declare variables not at the beginning of the scope?

Comment (ProfessorKaos64) [inline Quake/Makefile]: [I haven't seen C89 added to a project in a long while](https://i.imgflip.com/wppxk.jpg)

Comment (sezero) [inline Quake/Makefile]: On 9/13/16, Axel Gneiting <notifications@github.com> wrote:
>> -CFLAGS += -std=gnu99
>
> Are you sure this is fine? There are still plenty of functions where I
> declare variables not at the beginning of the scope?

Yes, however without an explicit -ansi or similar, gcc extensions
are in effect, so unless someone attempts using gcc2 (which btw
does not support -std=gnu99), you are fine.

**Pull request merged**